### PR TITLE
add automation tests

### PR DESCRIPTION
**Pull request by ksimon1**
This PR adds basic functional test, which deploys minishift, builds image for kubevirt-node-labeller and tries to deploy it into cluster. Then it checks if minishift node has cpu model, cpu feature labels. Due to travis restriction, there is no access to kvm device, so libvirt has to use --virttype qemu. This test tests only if labeller updates node with new labels (it doesn't check if e.g. cpu models are really supported by host or config map for cpu plugin works correctly...)
This PR add ansible for generating release manifests.

### CODE REVIEW

**Comment by ksimon1** [comment]
@fromanirh, @MarSik, @yanirq please review. This is first version of functional test for labeller. We can probably reuse deployment of minishift in ssp operator.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+  - stage: Functional test
+    name: Test kubevirt-node-labeller bundle
+    before_script:
+    - git clone https://github.com/fabiand/traviskube.git
+    - mv traviskube ci 
+    - sudo mount --make-rshared /
+    - bash -x ci/prepare-host minishift
+    - bash -x ci/prepare-host oc
+    - bash -x ci/start-cluster minishift
+    - bash -x ci/deploy-kubevirt minishift v0.16.0
```
Comment (MarSik) [inline .travis.yml]: There is 0.17.0 now I think

Comment (MarSik) [reply .travis.yml]: But do you need kubevirt at all? I mean this is an isolated labeller component so checking whether the nodes got the labels might be enough..

Comment (ksimon1) [reply .travis.yml]: you are right, Since travis doesn't have access to kvm, so kubevirt is not necessary

Comment (ksimon1) [reply .travis.yml]: for some unknown reasons, libvirt doesn't show any output of domcapabilities without kubevirt. I will have to investigate it.

**Comment by ksimon1** [comment]
Closing in favour of https://github.com/kubevirt/node-labeller/pull/12 (github has some issue, that PRs opened before repos transfer, don't reflect new changes on branches)

**Pull request closed without merging**